packages/backend-next/src/index.ts

```diff
@@ -45,4 +46,7 @@ backend.add(import('@backstage/plugin-search-backend/alpha'));
 backend.add(import('@backstage/plugin-techdocs-backend/alpha'));
 backend.add(import('@backstage/plugin-todo-backend'));
 
+// jenkins
+backend.add(jenkinsPlugin());
+
```

Comment:
Heads-up that all comments & empty lines were removed prior. Probably it make sense here as well.

```suggestion
backend.add(jenkinsPlugin());
```

Reply:
updated as per suggestion